setup.py

```diff
@@ -1,20 +1,17 @@
 from setuptools import setup
 
-readme = open('README.rst').read()
 history = open('CHANGES.rst').read().replace('.. :changelog:', '')
 
 setup(
-    name='sqlalchemy-redshift',
+    name='gx-sqlalchemy-redshift',
     version='0.8.15.dev0',
-    description='Amazon Redshift Dialect for sqlalchemy',
-    long_description=readme + '\n\n' + history,
+    description='Forked Amazon Redshift Dialect for sqlalchemy',
+    long_description=history,
     long_description_content_type='text/x-rst',
```

Comment:
We should try updating this to `text/markdown` and see if we can get the readme to render correctly?